<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Home</class>
 <widget class="QWidget" name="Home">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1294</width>
    <height>837</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QCheckBox" name="checkBox_saltWater">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(66, 118, 253);
color:white;</string>
   </property>
   <property name="text">
    <string>Salt Water Fish</string>
   </property>
  </widget>
  <widget class="QListWidget" name="educationBanner">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="217">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="63">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <item>
    <property name="text">
     <string>
                 	Home
</string>
    </property>
    <property name="background">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>66</red>
       <green>118</green>
       <blue>253</blue>
      </color>
     </brush>
    </property>
    <property name="foreground">
     <brush brushstyle="DiagCrossPattern">
      <color alpha="255">
       <red>250</red>
       <green>250</green>
       <blue>250</blue>
      </color>
     </brush>
    </property>
   </item>
  </widget>
  <widget class="QCheckBox" name="checkBox_freshWater">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>230</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(66, 118, 253);
color:white;

</string>
   </property>
   <property name="text">
    <string>Fresh Water Fish</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_smallTanks">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>330</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(66, 118, 253);
color: white</string>
   </property>
   <property name="text">
    <string>Small Tank Fish</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_bigTanks">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>280</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(66, 118, 253);
color: white;</string>
   </property>
   <property name="text">
    <string>Big Tank Fish</string>
   </property>
  </widget>
  <widget class="QLabel" name="fishIcon">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>71</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Resources.qrc">:/images/fish-logo.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>40</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>35</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Fish Buddies</string>
   </property>
  </widget>
  <widget class="QPushButton" name="homePreference_button">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>50</y>
     <width>131</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Preference Setting</string>
   </property>
  </widget>
  <widget class="QPushButton" name="edu_button">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>380</y>
     <width>101</width>
     <height>32</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(66, 118, 253);
color:white;</string>
   </property>
   <property name="text">
    <string>Education</string>
   </property>
  </widget>
  <widget class="QPushButton" name="trade_button">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>530</y>
     <width>101</width>
     <height>32</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(66, 118, 253);
color:white;</string>
   </property>
   <property name="text">
    <string>Trade</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>530</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Resources.qrc">:/images/kindpng_1973521.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>380</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Resources.qrc">:/images/clipart4449882.png</pixmap>
   </property>
  </widget>
  <widget class="QScrollArea" name="add_a_post">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>230</y>
     <width>451</width>
     <height>221</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>449</width>
      <height>219</height>
     </rect>
    </property>
    <widget class="QLineEdit" name="title_input">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="body_input">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_salt">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Salt Water Fish</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_fresh">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>140</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Fresh Water Fish</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_bigTank">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Big Tank Fish</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_smallTank">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>170</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Small Tank Fish</string>
     </property>
    </widget>
    <widget class="QLabel" name="title_label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Title</string>
     </property>
    </widget>
    <widget class="QLabel" name="title_label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Body</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addPost_button">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>170</y>
       <width>131</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Add a post</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>170</y>
     <width>431</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="postLayout"/>
  </widget>
  <widget class="QPushButton" name="chat_button">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>430</y>
     <width>101</width>
     <height>32</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(66, 118, 253);
color:white;</string>
   </property>
   <property name="text">
    <string>Chat</string>
   </property>
  </widget>
  <widget class="QPushButton" name="friend_button">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>480</y>
     <width>101</width>
     <height>32</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(66, 118, 253);
color:white;</string>
   </property>
   <property name="text">
    <string>Friends</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>480</y>
     <width>58</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Resources.qrc">:/images/clipart4273239.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>430</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Resources.qrc">:/images/NicePng_png-ballons_4121775.png</pixmap>
   </property>
  </widget>
  <widget class="QListWidget" name="banner">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>191</width>
     <height>431</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="217">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="63">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <item>
    <property name="text">
     <string>





AKA




















</string>
    </property>
    <property name="background">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
    <property name="foreground">
     <brush brushstyle="DiagCrossPattern">
      <color alpha="255">
       <red>250</red>
       <green>250</green>
       <blue>250</blue>
      </color>
     </brush>
    </property>
   </item>
  </widget>
  <zorder>banner</zorder>
  <zorder>checkBox_saltWater</zorder>
  <zorder>educationBanner</zorder>
  <zorder>checkBox_freshWater</zorder>
  <zorder>checkBox_smallTanks</zorder>
  <zorder>checkBox_bigTanks</zorder>
  <zorder>fishIcon</zorder>
  <zorder>label</zorder>
  <zorder>homePreference_button</zorder>
  <zorder>edu_button</zorder>
  <zorder>trade_button</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>add_a_post</zorder>
  <zorder>verticalLayoutWidget</zorder>
  <zorder>chat_button</zorder>
  <zorder>friend_button</zorder>
  <zorder>label_5</zorder>
  <zorder>label_4</zorder>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
